src/cpr_sdk/yql_builder.py

```diff
@@ -63,6 +63,12 @@ def build_search_term(self) -> str:
                     (userInput(@query_string)) 
                 )
             """
+        elif self.params.by_document_title:
+            return """
+                (
+                    (document_title_not_stemmed contains({stem: false}@query_string))
```

Comment:
why unstemmed over stemmed?

Reply:
whoops! wasn't thinking

Reply:
have [changed it to stemmed](https://github.com/climatepolicyradar/cpr-sdk/pull/209/commits/421d815151f726175e973a68c7037d6f2696190b) and tested locally